### PR TITLE
Added plot for venus transit for coordinate section

### PR DESCRIPTION
**Pull request by ehsteve**
The following code requires that @ayshih PR#3055 as well as PR#3056 which have not yet been accepted.

### CODE REVIEW

**Comment by ehsteve** [comment]
Also, why is the default colorable settings for 4500 look so bad? @wafels?

File: figures/venus_transit.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+sunpy_paper.setup_plot()
+
+# get the data of interest and save fits files
+map1 = map.Map(os.path.join(sunpy_paper.data_dir, 'aia.lev1.4500A_2012-06-05T23_30_34.90Z.image_lev1.fits'))
+
+# now get venus position
+venus = get_body_heliographic_stonyhurst('venus', map1.date, observer=map1.observer_coordinate)
+
+venus_hpc = venus.transform_to(frames.Helioprojective(observer=map1.observer_coordinate))
```
Comment (Cadair) [inline figures/venus_transit.py]:
```suggestion
```

File: figures/venus_transit.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+map1.plot(axes=ax1)
+map1.draw_limb()
+
+#x and y plotting limits in arcsec
+lims_arcsec = ((-800, 200)*u.arcsec, (00, 1000)*u.arcsec)
+lims_pix_ax1 = map1.world_to_pixel(SkyCoord(*lims_arcsec, frame=map1.coordinate_frame))
+ax1.set_xlim(lims_pix_ax1[0].value)
+ax1.set_ylim(lims_pix_ax1[1].value)
+
+ax1.plot_coord(venus_hpc, 'x', color='white', fillstyle='none')
```
Comment (Cadair) [inline figures/venus_transit.py]:
```suggestion
ax1.plot_coord(venus, 'x', color='white', fillstyle='none')
```

**Review by Cadair — COMMENTED** [review]
Couple of little tweaks I think will simplify it

File: figures/venus_transit.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+venus = get_body_heliographic_stonyhurst('venus', map1.date, observer=map1.observer_coordinate)
+
+venus_hpc = venus.transform_to(frames.Helioprojective(observer=map1.observer_coordinate))
+fig = plt.figure(figsize=(8, 11))
+ax1 = fig.add_subplot(1, 1, 1, projection=map1)
+map1.plot(axes=ax1)
+map1.draw_limb()
+
+#x and y plotting limits in arcsec
+lims_arcsec = ((-800, 200)*u.arcsec, (00, 1000)*u.arcsec)
```
Comment (Cadair) [inline figures/venus_transit.py]: I think:
```suggestion
lims_arcsec = ((-800, 200), (0, 1000))*u.arcsec
```

File: figures/venus_transit.py (hunk 2 above)
Comment (ehsteve) [reply figures/venus_transit.py]: This does not work if Venus is not converted to HPC. I assume it is supposed to.

Comment (Cadair) [reply figures/venus_transit.py]: I will investigate sometime when I get a chance.

Comment (Cadair) [reply figures/venus_transit.py]: https://github.com/astropy/astropy/pull/8927 fixes this btw.

**Pull request closed without merging**